package/readme.txt

```diff
@@ -36,6 +36,8 @@ New features:
 
 Improvements:
 - When sorting by any column, always start with ascending sort order
+- Improved Debug Console - now includes stacktraces and it can be opened in background (non-modal) with logs autorefresh
```

Comment:
please separate to three lines (support stacktraces, open in background, auto-refresh)